lib/topological_inventory/ansible_tower/receptor/response_worker.rb

```diff
@@ -0,0 +1,74 @@
+module TopologicalInventory::AnsibleTower
+  module Receptor
+    class ResponseWorker
+      def self.instance
+        @instance ||= self.new
```

Comment:
Eh not a huge fan of the global singleton, could you just keep an instance in the connection and pass it wherever else is needed?

Reply:
ok, will do

Reply:
:+1: thanks

Reply:
@agrare I refactored this singleton, but there is no place for instance variable. 
In this case singleton has reasonable usage, any other option doesn't improve stability or readability. So what exactly is the problem you have with singleton here?